pkg/action/build-uki.go

```diff
@@ -424,11 +431,23 @@ func (b *BuildUKIAction) ukify(sourceDir, cmdline string) error {
 // TODO: the efi file should come from the downloaded image, not from the
 // enki running OS.
 func (b *BuildUKIAction) sbSign(sourceDir string) error {
+	var systemdBoot string
+	var outputEfi string
+	if b.arch == "x86_64" || b.arch == "amd64" {
```

Comment:
I'd go one step further and introduce `isAmd64` something like that

Reply:
too much abstraction for a simple thing IMO....

Reply:
done

Reply:
I don't think it's too much ... otherwise when we want to change something we have to grep everywhere. I think it's nice to have a single source of truth ... maybe right now it feels like much but at least for me, when reading all this stuff again it simplifies it

Reply:
So.... A spec with the final data to consume? xD